app/controllers/downtimes_controller.rb

```diff
@@ -48,6 +48,11 @@ def downtime_params
     params[:downtime].permit([
       "artefact_id",
       "message",
+    ]).merge!(date_params)
```

Comment:
I don't like this at all. As the fields are now built in a more granular way (hour, min & day, month, year) rather than using the `time_select` and `date_select` helper methods, they need to be given unique field names. 

This means that they don't correspond with the `start_time` and `end_time` attrs on the `Downtime` object. As a result you can't call `f.select_hour` etc.

I've got round this by merging the params in the controller. A more optimal solution would be to create an intermediary object like a FormObject with the correct attrs, but i'm not sure if it's worth introducing a new pattern just for this. 

Wdyt?

Reply:
Could you specify a `prefix` of `downtime` for the `select_day` (etc) helpers (i.e. override the default `date` prefix)?  It also means none of the tests need updating.

I've put some suggested code for this in a branch: https://github.com/alphagov/publisher/compare/downtime-page-fixes-bruce?expand=1

Reply:
Amazing. I had no idea you could prefix the attr's. Will apply these changes to my code 👍 Thanks Bruce